mesh_tools/create_SCRIP_files/create_SCRIP_file_from_CISM_mesh.py

```diff
@@ -69,7 +87,18 @@
 # Dimensions
 fout.createDimension("grid_size", nx * ny)
 fout.createDimension("grid_corners", 4 )
-fout.createDimension("grid_rank", 1)
+
+##HH
+if int(options.gridRank) == 1:
+    print('grid rank is 1')
+    fout.createDimension("grid_rank", 1)
+elif int(options.gridRank) == 2:
+    print('grid rank is 2')
+    fout.createDimension("grid_rank", 2)
+else:
+    print('grid rank value is invalid: valid options are 1 or 2. Please try it again.')
+    exit()
```

Comment:
A more typical way to do this would be:
```suggestion
else:
    raise ValueError(f'grid rank value is invalid: valid options are 1 or 2 but {options.gridRank} was given.') 
```

Reply:
Oh, very cool. Got it! Thanks, Xylar :)